test/DFPerfScenarios/DFPerfScenariosV4.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net60</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
oh my. Was this even working before?

Reply:
I think we added this folder later and never put it in our build haha

Reply:
Let me just triple check then - is this now in the build? If so - where :)